### ISSUE TITLE
Importing pywinauto changes the fileopenbox UI

### ISSUE THREAD

**Issue by breadcat-rr** [robertlugg/easygui#200]
When importing `pywinauto` it makes the fileopenbox (and probably others, untested) to a less usable, worse UI.
e.g
```py
import pywinauto
import easygui
easygui.fileopenbox()
```
Expected UI.
![Expected UI](https://user-images.githubusercontent.com/40906448/163580048-fbf98711-cae1-4326-a668-f81c8f57cc37.png)

The UI that is actually presented.
![pywinauto hijacked UI](https://user-images.githubusercontent.com/40906448/163580106-c2ad80ae-1043-4ba0-a1b3-ce3bed41cfab.png)

**Comment by zadacka** [comment]
Strange - I can't reproduce this. Did something else change in between the before/after screenshots? If you comment out the pywinauto import do you still see the unexpected (light) dialog?

**Comment by breadcat-rr** [comment]
For me at least, when I comment out `pywinauto` it shows back to the expected UI

**Comment by zadacka** [comment]
Hmm, what versions of easygui and pywinauto, and and which Windows OS are
you using? I’ll have another shot at reproducing it..

**Comment by breadcat-rr** [comment]
easygui:
`Version: 0.98.2`

pywinauto
`Version: 0.6.8`

OS: Windows 10 Pro
Version: 20H2 
OS Build: 19042.1348

**Comment by zadacka** [comment]
I suspect that something in pywinauto is changing the tkinter ttk theme from dark to light, but I haven't identified the relevant bit yet.  My windows theme is light and I wasn't aware of anything in easygui that changed it.

What version of tkinter are you using? It's really strange that something in pywinauto is changing the default state. My understanding was that styles would be applied to a root tkinter box - so my expectation was the your easygui box would have a different root and so a different (default) style irrespective of whatever pywinauto might do

**Comment by breadcat-rr** [comment]
I'm using tkinter version `8.6`. I don't think its changed at all because I never update my packages lol

**Comment by hrpzcf** [comment]
Maybe it's a bug of pywinauto, edit `pywinauto/__init__.py` line 83, change `return com_init_mode` to `return 2`, it should work.

**Comment by breadcat-rr** [comment]
Yep, that works, Thanks!. Also found out that `easygui.diropenbox()` just hangs if pywinauto is unchanged.
